### PR TITLE
fix anysetter to properly work for deserialization

### PR DESCRIPTION
**Pull request by wistefan**
Fix for the any-setter to work on deserialization.

### CODE REVIEW

File: src/main/resources/Micronaut/modelPojo.mustache [hunk 1]
```diff
@@ -85,9 +85,12 @@ public {{#discriminator}}abstract {{/discriminator}}class {{classname}}{{#parent
 	}
 
 	@com.fasterxml.jackson.annotation.JsonAnySetter
-	public {{classname}} setAdditionalProperties({{{vendorExtensions.additionalPropertiesMap}}} newAdditionalProperties) {
-		this.additionalProperties = newAdditionalProperties;
-		return this;
+	public {{classname}} setAdditionalProperties(java.lang.String propertyKey, java.lang.Object value) {
+		if (this.additionalProperties == null) {
```
Comment (axdotl) [inline src/main/resources/Micronaut/modelPojo.mustache:89]:
```suggestion
	if (this.additionalProperties == null) {
```

File: src/main/resources/Micronaut/modelPojo.mustache [hunk 2]
```diff
@@ -85,10 +85,19 @@ public {{#discriminator}}abstract {{/discriminator}}class {{classname}}{{#parent
...
+	public {{classname}} setAdditionalProperties(java.lang.String propertyKey, java.lang.Object value) {
+	    if (this.additionalProperties == null) {
+            this.additionalProperties = new java.util.HashMap<>();
+        }
+        this.additionalProperties.put(propertyKey, value);
+        return this;
 	}
+
+	public {{classname}} setAdditionalProperties({{{vendorExtensions.additionalPropertiesMap}}} newAdditionalProperties) {
+    		this.additionalProperties = newAdditionalProperties;
```
Comment (axdotl) [inline src/main/resources/Micronaut/modelPojo.mustache:97]: What's wrong w/ your formatter 🤣 
```suggestion
	this.additionalProperties = newAdditionalProperties;
```

File: src/main/resources/Micronaut/modelPojo.mustache [hunk 3]
```diff
@@ -85,10 +85,19 @@ public {{#discriminator}}abstract {{/discriminator}}class {{classname}}{{#parent
...
+    		return this;
```
Comment (axdotl) [inline src/main/resources/Micronaut/modelPojo.mustache:98]:
```suggestion
	return this;
```

File: src/main/resources/Micronaut/modelPojo.mustache (hunk 2 above)
Comment (wistefan) [reply src/main/resources/Micronaut/modelPojo.mustache:97]: > What's wrong w/ your formatter rofl
It does not consider mustache as java...

**Review by stephanschnabel — APPROVED** [review]

**Pull request merged**